CMakeLists.txt

```diff
@@ -4,6 +4,10 @@ project( googletest-distribution )
 
 enable_testing()
 
+include(GNUInstallDirs)
+
+option(ENABLE_GOOGLETEST_INSTALL "Skip installation" OFF )
```

Comment:
Should I delete the space on the next two lines also then?  I'm just trying to fit with what the format already was.  Or should it only have  a space if there's also a comment on the line?